Gemfile

```diff
@@ -5,7 +5,7 @@ gem 'activerecord', ENV['AR'] ? ENV['AR'].split(",") : [">= 4.2.7", "< 5.3.0"]
 gem 'railties', ENV['AR'] ? ENV['AR'].split(",") : [">= 4.2.7", "< 5.3.0"]
 
 group :dev do
-  gem 'sqlite3'
+  gem 'sqlite3', '~> 1.3.6'
```

Comment:
Any reason you're pinning sqlite3 at 1.3.6?

Reply:
travis build fails if you didn't add sqlite3 version